test/dynamo/test_modules.py

```diff
@@ -1818,6 +1823,26 @@ def forward(self, x):
         self.assertEqual(eager_res, optim_res)
         self.assertEqual(cnt.frame_count, 1)
 
+    def test_torch_save_error(self):
+        mod = BasicModule()
+        opt_model = torch.compile(mod)
+
+        with tempfile.TemporaryDirectory() as temp_dir:
+            file_path = os.path.join(temp_dir, "foo.pt")
+            with pytest.raises(TypeError) as excinfo:
```

Comment:
No need for pytest here. You can use `self.assertRaisesRegex()` and provide a regex for what you want to match in the error message.